src/wagtail_live/models.py

```diff
@@ -51,12 +51,69 @@ def last_update_timestamp(self):
 
         return self.last_updated_at.timestamp()
 
+    def __init__(self, *args, **kwargs):
+        """Add extra attributes to track changes made in the admin interface."""
+
+        super().__init__(*args, **kwargs)
+        self._previous_posts = self.live_posts
+        self._synced = self._has_changed = False
+
+    def save(self, *args, **kwargs):
+        result = super().save(*args, **kwargs)
+
+        # Update extra attributes when the page is saved
+        self._previous_posts = self.live_posts
+        self._synced = self._has_changed = False
+        return result
+
     def clean(self):
-        """Update last_updated_at when the page is modified on the admin interface."""
+        """Update `last_updated_at` and `modified` when the page is modified
+        via the admin interface."""
 
-        self.last_updated_at = timezone.now()
         super().clean()
 
+        if not self._synced:
```

Comment:
**Question:** I wonder if it makes sense to extract this into a separate method that we then call from `clean()`?